src/legacy/core_plugins/kibana/server/tutorials/apm/instructions/apm_agent_instructions.js

```diff
@@ -304,52 +304,66 @@ export const createRackAgentInstructions = (apmServerUrl = '', secretToken = '')
 export const createJsAgentInstructions = (apmServerUrl = '') => [
   {
     title: i18n.translate('kbn.server.tutorials.apm.jsClient.enableRealUserMonitoring.title', {
-      defaultMessage: 'Enable Real User Monitoring support in the APM server',
+      defaultMessage: 'Enable Real User Monitoring support in APM server',
     }),
     textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.enableRealUserMonitoring.textPre', {
-      defaultMessage: 'Please refer to [the documentation]({documentationLink}).',
-      values: { documentationLink: '{config.docs.base_url}guide/en/apm/server/{config.docs.version}/rum.html' },
+      defaultMessage: 'APM Server disables RUM support by default. See the [documentation]({documentationLink}) \
+for details on how to enable RUM support.',
+      values: { documentationLink: '{config.docs.base_url}guide/en/apm/server/{config.docs.version}/configuration-rum.html' },
     }),
   },
   {
-    title: i18n.translate('kbn.server.tutorials.apm.jsClient.install.title', {
-      defaultMessage: 'Install the APM agent',
-    }),
-    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.install.textPre', {
-      defaultMessage: 'Install the APM agent for JavaScript as a dependency to your application:',
-    }),
-    commands: [`npm install @elastic/apm-rum --save`],
-  },
-  {
-    title: i18n.translate('kbn.server.tutorials.apm.jsClient.configure.title', {
-      defaultMessage: 'Configure the agent',
+    title: i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.title', {
+      defaultMessage: 'Set up the Agent as a dependency',
     }),
-    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.configure.textPre', {
-      defaultMessage: 'Agents are libraries that run inside of your application.',
+    textPre: i18n.translate('kbn.server.tutorials.apm.jsClient.installDependency.textPre', {
+      defaultMessage: 'You can install the Agent as a dependency to your application with \
+`npm install @elastic/apm-rum --save`.\n\n\
```

Comment:
We have different packages for React and Angular, Should we expand the installation process to include those as well?

Reply:
I see two options:

1. Expanding the installation tutorial and including them here as well.
2. Leaving them in the docs and linking from this tutorial.

I'd rather do number two, but if you feel strongly about #1, we can chat.

Reply:
I updated the screenshot above. Let me know what you think.

Reply:
Yeah 2nd option seems good to be, less confusing for new users.